aleph-client/src/lib.rs

```diff
@@ -1,3 +1,7 @@
+// Needed for the generic `impl TryFrom<ConvertibleValue> for Option<T>`.
+#![feature(auto_traits)]
+#![feature(negative_impls)]
```

Comment:
Maybe we should hide all of this behind a feature? That way if someone doesn't need the particular conversion they can disable the feature and not be exposed to these dragons. Not sure if that's considered good practice in rustland.

Reply:
That's a good point but I can't imagine people wanting `aleph-client` and, at the same time, not wanting the conversion from `Value` to `Option<T>` 🤷

Reply:
I guess it depends on the contract they're interacting with? I don't quite know what's the impact of enabling these features. Is there any chance this blows up someone's compiler? If that chance is low, then probably no point in doing that. Maybe @mike1729 could opine.